syntax = "proto3";
package drivebrain_core_msgs;

message SpeedControlOut {
    float pos_torque_limit_nm = 1;
    float neg_torque_limit_nm = 2;
    float desired_rpm = 3;
}

message SpeedControlIn {
    float accel_percent = 1;
    float brake_percent = 2;
}